packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2660,6 +2660,11 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
   BuildOwner get owner => _owner;
   BuildOwner _owner;
 
+  /// Whether the element is active.
+  ///
+  /// A element will be active if it is mounted and attached to the element
+  /// tree.
+  bool get isActive => _active;
```

Comment:
We should not have this. Once an object is disposed, it should never be referenced, so there should be no need for it.